src/ray/protobuf/gcs.proto

```diff
@@ -264,6 +269,8 @@ message JobConfig {
   string ray_namespace = 5;
   // An opaque kv store for job related metadata.
   map<string, string> metadata = 6;
+  // The default lifetime of actors with lifetime not specified explicitly at runtime.
```

Comment:
The comment in ray.default.conf is more clear. Can you change this to the same as that one?